static/app/components/badge/tag.tsx

```diff
@@ -18,6 +18,7 @@ import type {Color} from 'sentry/utils/theme';
 import theme from 'sentry/utils/theme';
 
 export interface TagProps extends React.HTMLAttributes<HTMLSpanElement> {
+  borderStyle?: 'solid' | 'dashed' | 'dotted' | 'double' | 'none';
```

Comment:
There are [more](https://www.w3schools.com/css/css_border.asp) border styles than this, but I just added the 2d ones that I thought were relevant - feel free to correct me if there's a better way to do this

Reply:
Removed double because it doesn't appear to do anything. 
Removed none because it just becomes text at that point